SessionUtilitiesKit/JobRunner/JobRunner.swift

```diff
@@ -1901,6 +1901,7 @@ public extension JobRunner {
     }
     
     static func appDidBecomeActive(using dependencies: Dependencies = Dependencies()) {
+        SNLog("[JobRunner] appDidBecomeActive")
```

Comment:
Just wanted to check whether we want to keep this log? (Just thinking that it's going to appear every time the app returns from the background and we already have `"[AppDelegate] applicationWillEnterForeground."` and `"[AppDelegate] applicationDidBecomeActive."` so want to avoid adding another startup log if it's not going to add much value)

Reply:
I'll remove it, it is just for some testing